modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala

```diff
@@ -63,6 +68,13 @@ object NewPullRequestData {
         |""".stripMargin.trim
   }
 
+  def fromTo(update: Update, branchCompareUrl: Option[String]): String =
+    branchCompareUrl match {
+      case None => s"from ${update.currentVersion} to ${update.nextVersion}"
+      case Some(compareUrl) =>
+        s"[from ${update.currentVersion} to ${update.nextVersion}](${compareUrl})"
```

Comment:
`from ${update.currentVersion} to ${update.nextVersion}` is duplicated here.

Reply:
Extracted the duplicates in https://github.com/fthomas/scala-steward/pull/814/commits/06e5f33d68e25ceb25f57101dc057c8b40f79cc1